addons/project/static/src/components/subtask_kanban_list/subtask_kanban_list.js

```diff
@@ -75,15 +88,25 @@ export class SubtaskKanbanList extends Component {
     }
 
     async _onSubtaskCreateNameChanged(name) {
-        await this.orm.create("project.task", [{
-            display_name: name,
-            parent_id: this.props.record.resId,
-            project_id: this.props.record.data.project_id[0],
-            user_ids: this.props.record.data.user_ids.resIds,
-        }]);
-        this.subtaskCreate.open = false;
-        this.subtaskCreate.name = "";
-        this.props.record.load();
+        if (name.trim() === "") {
+            this.notification.add(
+                markup(`<ul><li>${_t("Display Name")}</li></ul>`),
```

Comment:
We need to escape the translation part. 🙂 

first we need to import the escape function

```js
import { escape } from "@web/core/utils/strings";
```

and then we have to do this: 
```suggestion
                markup(`<ul><li>${escape(_t("Display Name"))}</li></ul>`),
```

Reply:
done the changes. :+1: